pkg/core/resources/manager/manager.go

```diff
@@ -168,7 +168,7 @@ func Upsert(ctx context.Context, manager ResourceManager, key model.ResourceKey,
 			return err
 		}
 		err := upsert(ctx)
-		if store.IsResourceConflict(err) || store.IsResourceAlreadyExists(err) {
```

Comment:
why ResourcesAlreadyExist is gone?

Reply:
The reason `IsResourceAlreadyExists` was added is that you may have a conflict when you first create a resource from two places